projects/helper/unwrapLPs.js

```diff
@@ -249,11 +249,12 @@ async function unwrapUniswapLPs(balances, lpPositions, block, chain='ethereum',
         const token1 = (await tokens1).output.find(call=>call.input.target === lpToken).output
         const supply = (await lpSupplies).output.find(call=>call.input.target === lpToken).output
         const {_reserve0, _reserve1} = (await lpReserves).output.find(call=>call.input.target === lpToken).output
-
-        const token0Balance = BigNumber(lpPosition.balance).times(BigNumber(_reserve0)).div(BigNumber(supply))
-        sdk.util.sumSingleBalance(balances, await transformAddress(token0.toLowerCase()), token0Balance.toFixed(0))
-        const token1Balance = BigNumber(lpPosition.balance).times(BigNumber(_reserve1)).div(BigNumber(supply))
-        sdk.util.sumSingleBalance(balances, await transformAddress(token1.toLowerCase()), token1Balance.toFixed(0))
+        if (token0 && token1 && supply > 0 && supply && lpPosition.balance) {
```

Comment:
this extra check could generate some issues, problem is that the RPC that we are using might cause some calls to fail incorrectly (RPC returns call as failed but it does't actually fail, it's just that the rpc failed for some other reason). With the current code this will lead to the adapter breaking, however, with this new addition the adapter will just ignore this LP position, so it will report incorrect numbers